src/abyssal_tome/app.py

```
@@ -1,729 +1,88 @@
+    class Config:
+        orm_mode = True
+
+# --- FastAPI App ---
+app = FastAPI()
+
+# --- Dependency ---
+def get_db():
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
```

Ruff (0.11.9)

Missing return type annotation for public function `get_db`

(ANN201)